config/multi-cluster-monitor.properties

```diff
@@ -12,12 +12,13 @@
 # each cluster in the pipeline. The "produce.service.props" should use the first cluster and
 # the "consume.service.props" should use the last cluster in the pipeline.
 
+# Produce service: Configure Produce Service to produce to the first cluster of the pipeline
```

Comment:
comments `repositioned` to the top since it was interfering with the properties compilation.